atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/TimestampStateStore.java

```diff
@@ -143,13 +147,16 @@ Optional<CommitInfo> getCommitInfo(long startTimestamp) {
         return getTimestampInfo(startTimestamp).flatMap(TimestampVersionInfo::commitInfo);
     }
 
-    @VisibleForTesting
-    TimestampStateStoreState getStateForTesting() {
+    @Unsafe
+    TimestampStateStoreState getStateForDiagnostics() {
         // This method doesn't need to read a thread-safe snapshot of timestampMap and livingVersions
         SortedSetMultimap<Sequence, StartTimestamp> living = TreeMultimap.create();
-        livingVersions.forEach(living::putAll);
+        for (Entry<Sequence, NavigableSet<StartTimestamp>> entry : livingVersions.entrySet()) {
+            living.putAll(entry.getKey(), new TreeSet<>(entry.getValue()));
+        }
```

Comment:
Why not this -> `livingVersions.forEach((key, value) -> living.putAll(key, new TreeSet<>(value)));`

Reply:
Changed!